profiles/models.py

```diff
@@ -2,8 +2,7 @@
 from django.db import models
 from django.conf import settings
 
-# This matches ImageField.max_length
-MAX_IMAGE_FIELD_LENGTH = 100
+MAX_IMAGE_FIELD_LENGTH = 1024
```

Comment:
Why this other size?

Reply:
It's less than the max size which is about 2k, and I think it's large enough to handle the URLs we'll probably use